src/entities/Posts.ts

```diff
@@ -15,6 +15,7 @@ export class Posts {
 	date_created: Date;
 
 	@ManyToOne(type => Subreddits, subreddit => subreddit.posts)
+	@Column()
```

Comment:
Why? You already create column in database with ManyToOne notation. Why we need another column in database table?